src/test/java/org/fcrepo/importexport/exporter/ExporterTest.java

```diff
@@ -65,7 +65,7 @@
     private Config args;
     private Config binaryArgs;
     private Config metadataArgs;
-
+    private String exportDirectory = "target/export";
```

Comment:
This is very minor, but for readability, please add an empty line before `@Before`.